components/SearchSandbox/template.js

```diff
@@ -32,98 +32,105 @@ import {
 import 'antd/dist/antd.css';
 ${
 		showTree
-			? `import ExpandCollapse from 'react-expand-collapse';
-
-				import './styles.css';
-
-				const { TreeNode } = Tree;
-
-				const renderAsTree = (res, key = '0') => {
-					if (!res) return null;
-					const iterable = Array.isArray(res) ? res : Object.keys(res);
-					return iterable.map((item, index) => {
-						const type = typeof res[item];
-						if (type === 'string' || type === 'number') {
-							return (
-								<TreeNode
-									title={
-										<div>
-											<span>{item}:</span>&nbsp;
-											<span dangerouslySetInnerHTML={{ __html: res[item] }} />
-										</div>
-									}
-									key={key + "-" + (index + 1)}
-								/>
-							);
-						}
-						const hasObject = (res[item] === undefined && typeof item !== 'string');
-						const node = hasObject ? item : res[item];
-						return (
-							<TreeNode
-								title={typeof item !== 'string' ? 'Object' : '' + (node || Array.isArray(res) ? item : item + ': null')}
-								key={key + "-" + (index + 1)}
-							>
-								{renderAsTree(node, key + "-" + (index + 1))}
-							</TreeNode>
-						);
-					});
-				};
-
-				function onData(res) {
-					return (
-						<div className="list-item" key={res._id}>
-							<ExpandCollapse
-								previewHeight="390px"
-								expandText="Show more"
-							>
-								{
-									<Tree showLine>
-										{renderAsTree(res)}
-									</Tree>
-								}
-							</ExpandCollapse>
+			? `
```

Comment:
It'd best if we put this static code in some file (assigned to a constant) and import it from there.